example.go

```diff
@@ -0,0 +1,23 @@
+package booleans
+
+
+//canExecuteFastAttack indicates if the knight is awake.
+func canExecuteFastAttack(knightIsAwake bool) bool {
```

Comment:
Should the functions in this file all start with an uppercase letter? That is the convention used in the other exercises.

Reply:
Yes, they should. This change is already in https://raw.githubusercontent.com/exercism/v3/master/languages/go/exercises/concept/booleans/.meta/example.go